src/vs/workbench/parts/lib/node/configVariables.ts

```diff
@@ -25,14 +25,16 @@ export class ConfigVariables extends SystemVariables {
 	}
 
 	protected resolveString(value: string): string {
+		const originalValue = value;
 		value = super.resolveString(value);
 
 		let regexp = /\$\{config\.(.*?)\}/g;
 		return value.replace(regexp, (match: string, name: string) => {
 			let config = this.configurationService.getConfiguration();
 			let newValue = new Function('_', 'try {return _.' + name + ';} catch (ex) { return "";}')(config);
```

Comment:
Can we simplify this line such that it is more readable? Also new Function syntax is not our style - it is not used anywhere else in our code base so I would suggest to change this